script.service.ltvpl/resources/skins/Default/720p/BusyDialog2.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<window>
+	<animation effect="fade" end="100" time="560">WindowOpen</animation>
+	<animation effect="fade" start="100" end="0" time="240">WindowClose</animation>
+	<zorder>2</zorder>
+	<controls>
+		<control type="group">
+			<visible>String.IsEmpty(Window(Home).Property(script.cinemavision.running))</visible>
+			<animation effect="fade" time="400">VisibleChange</animation>
+			<visible>!Window.IsActive(startup)</visible>
+			<control type="image">
+				<texture>colors/black.png</texture>
+				<include>FullScreenDimensions</include>
+				<animation effect="fade" start="100" end="70" time="0" condition="true">Conditional</animation>
+				<animation effect="fade" start="100" end="0" time="240" condition="Window.IsVisible(fullscreenvideo)">Conditional</animation>
+			</control>
+			<control type="group">
+				<depth>DepthMax</depth>
```

Comment:
there is no guarantee `DepthMax` is defined in the skin you're using

Reply:
removed